composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0",
-        "orchestra/testbench": "~3.0"
+        "orchestra/testbench-browser-kit": "~3.4"
```

Comment:
Why this needs to be changed? Looks like testbench repo has support for 5.4.
https://github.com/orchestral/testbench